Resources/Prototypes/_RMC14/Entities/Objects/Devices/whistle.yml

```diff
@@ -1,19 +1,25 @@
 ﻿- type: entity
-  parent: Whistle
+  parent: BaseItem
   id: RMCWhistle
   name: whistle
   description: A metal pea-whistle. Can be blown while held, or worn in the mouth
   components:
   - type: Sprite
     sprite: _RMC14/Objects/Devices/whistle.rsi
+    state: whistle
+  - type: Item
+    size: Tiny
   - type: Clothing
     slots:
       - mask
       - neck
   - type: EmitSoundOnUse
     sound:
       collection: TrenchWhistle
+  - type: EmitSoundOnAction
+    sound:
+      collection: TrenchWhistle
   - type: UseDelay
     delay: 10
-  - type: Whistle
+  - type: RMCWhistle
     distance: 15 # Let your enemies hear the sound of death approaching
```

Comment:
You don't need to modify upstream's whistle, we only use the downstream one with ID RMCWhistle

Reply:
Added the base Whistle component back to the prototype, the RMCWhistle component is now solely responsible for adding the action and for synchronizing the delays between in-hand and action usage